components/lib/inputnumber/InputNumber.js

```diff
@@ -1067,6 +1078,7 @@ export const InputNumber = React.memo(
                     onKeyPress={onInputKeyUp}
                     onInput={onInput}
                     onClick={onInputClick}
+                    onMouseDown={onInputMouseDown}
```

Comment:
Should this be `onPointerDown` instead so it works for touch devices as well?

Reply:
Thanks! I learned